src/mus2midi.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+#ifndef __MUS2MIDI_H__
```

Comment:
It is not worthy to have a header containing only one function prototype. Declare in `syscall_sdl.c` directly.